### PR TITLE
Adds Fastlane setup

### PR DESCRIPTION
**Pull request by mcousillas6**
Adds basic fastlane setup, with lanes for each target and some basic actions (version bump, automatic changelogs, etc.).

Check the new readme section for more details

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+# CONFIG VARIABLES
+username = 'toptiertest@gmail.com' # The apple id that you are using to admin the certificates eg: toptiertest@gmail.com
+xcodeproj = 'ios-base.xcodeproj' # The xcodeproj name eg: ReactNativeBase.xcodeproj
+workspace =  'ios-base.xcworkspace' # The xworkspace name here eg: ReactNativeBase.xcworkspace
```
Comment (pMalvasio) [inline fastlane/Fastfile]: Maybe we can create a new issue to adapt the swift init task to change these vars too. WDYT?

Comment (glm4) [reply fastlane/Fastfile]: How the init task will be affected by this? Am I missing something or this is only for release phase? @mcousillas6 ?

Comment (pMalvasio) [reply fastlane/Fastfile]: We just need to check if we need to update the init file to change the `ios-base` string in this file too.

Comment (mcousillas6) [reply fastlane/Fastfile]: Already checked, it's working just fine 👌

**Comment by CamilaMoscatelli** [comment]
Great work! 👏

**Pull request merged**